src/ex-gen/Program.cs

```diff
@@ -200,6 +200,7 @@ private static void ExportCsv(Database database, string path)
                 writer.Write("Namespace");
                 writer.Write("Type");
                 writer.Write("Member");
+                writer.Write("Site");
```

Comment:
I've no problem with emitting the data here, but we should have a global readonly field that controls this, defaulted to `false`.

The reason being that the file `etc\Exception.csv` that is written by this tool will be embedded as a resource in my analyzer. Adding the site the exception occurs resulted in a 60% increase of the file, but the data isn't useful to the analyzer.